File: Data Structures & Algorithms/max-water-container/submission-0.java
class Solution {
    public int maxArea(int[] heights) {
        int globalMax = 0;
        int i = 0;
        while (i < heights.length - 1) {
            
            int j = i + 1;
            while (j < heights.length) {
                System.out.println("(i, j): " + i + ", " + j);
                int curArea = (j - i) * Math.min(heights[i], heights[j]);
                System.out.println("curArea: " + curArea);
                globalMax = Math.max(curArea, globalMax);
                System.out.println("globalMax: " + globalMax);
                j++;
            }
            i++;
        }
        return globalMax;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        globalMax = 0
        localMax = 0
        for i in range(len(nums)):
            if nums[i] == 1:
                localMax += 1
            else:
                globalMax = max(globalMax, localMax)
                localMax = 0
        return max(localMax, globalMax)